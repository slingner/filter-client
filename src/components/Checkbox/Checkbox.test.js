import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import CheckBox from './Checkbox';

describe(`Checkbox component`, () => {
  const props = {
  }

  it('renders the checkbox by default', () => {
    const wrapper = shallow(<CheckBox />)
    expect(toJson(wrapper)).toMatchSnapshot()
  })

  it('renders the checkbox from props', () => {
    const wrapper = shallow(<CheckBox {...props} />)
    expect(toJson(wrapper)).toMatchSnapshot()
  })
})