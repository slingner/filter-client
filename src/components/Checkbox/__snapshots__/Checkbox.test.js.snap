// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Checkbox component renders the checkbox by default 1`] = `
<div
  className="checkbox_container"
>
  <div
    className="FlavorFilter"
  >
    Flavor Filter
  </div>
</div>
`;

exports[`Checkbox component renders the checkbox from props 1`] = `
<div
  className="checkbox_container"
>
  <div
    className="FlavorFilter"
  >
    Flavor Filter
  </div>
</div>
`;
